pkg/api/componentreadiness/query/querygenerators.go

```diff
@@ -367,6 +362,19 @@ func BuildComponentReportQuery(
 		})
 	}
 
+	// filter by test properties
+	if len(reqOptions.TestCapabilities) > 0 {
+		// include if there is any intersection between the arrays: capabilities filter and test capabilities
+		queryString += ` AND EXISTS(select 1
+		                            from UNNEST(@TestCapabilities) AS tcap
+		                            WHERE tcap in UNNEST(cm.capabilities)
```

Comment:
Does this work across components? Can you select multiple capabilities from different components this way?